### PR TITLE
fix(cluster_k8s): make scheme inline with current operator scheme

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/BBFkfUHO/2787-k8s-update-operator-scheme

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by dkropachev** [comment]
Verified - https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/dmitry-operator-byo-5/74/

File: sdcm/k8s_configs/provisioner/values.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ common:
   namespace: default
   #
   # Defines whether to create provisioner namespace
-  #
+  #
```
Comment (enaydanov) [inline sdcm/k8s_configs/provisioner/values.yaml:15]: Need to remove trailing space here and below.

**Review by enaydanov — CHANGES_REQUESTED** [review]

File: sdcm/k8s_configs/provisioner/values.yaml (hunk 1 above)
Comment (dkropachev) [reply sdcm/k8s_configs/provisioner/values.yaml:15]: This is copy paste from their repo, not sure we should do that

Comment (dkropachev) [reply sdcm/k8s_configs/provisioner/values.yaml]: fixed

**Review by enaydanov — APPROVED** [review]

**Pull request merged**